test/hotspot/gtest/nmt/test_nmt_startup_data.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#ifndef TEST_NMT_STARTUP_DATA_HPP
+#define TEST_NMT_STARTUP_DATA_HPP
+
+#include "stddef.h"
+
+typedef struct DataStruct
+{
+  long time;
+  long pointer;
+  long pointer_prev;
+  long frame1;
+  long frame2;
+  long frame3;
+  long frame4;
+  size_t requested;
+  size_t actual;
+  int flags;
+  int active;
+  const char* flags_name;
+}
+DataStruct;
+
```

Comment:
`{` to be moved one line above.
`}` to be on the same line as struct name.

Reply:
Fixed.